<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleProperty</class>
 <widget class="QDialog" name="ParticleProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Particle_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Particle ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Neighbor_Number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neighbor Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Density">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Density</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Velocity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Velocity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Constrain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Constrain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Viscosity_Force">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Viscosity Force</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Pressure_Force">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pressure Force</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_External_Force">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>External Force</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Damping_Force">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Damping Force</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParticleProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParticleProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
